src/js/_enqueues/wp/mce-view.js

```diff
@@ -643,16 +643,6 @@
 					}, 3000 );
 				}
 
-				function reload() {
```

Comment:
Rationale for removal: https://core.trac.wordpress.org/ticket/55491#comment:15

Reply:
I tried reproducing the issue in Firefox according to the description of Core-38511 and I was unable.

Reply:
I've reported this to the 7-year old ticket: https://core.trac.wordpress.org/ticket/38511#comment:27

Reply:
@azaozz also please weigh in on whether we can remove this code.

Reply:
Given the age of this bug fix, it should be fine to remove. If we discover that this still creates an issue for Firefox users, we can always restore the fix.

Reply:
Yep, super old browser specific fix. LGTM to remove too.